topics/04_oop_basic/hw/diamond_problem.py

```diff
@@ -0,0 +1,49 @@
+"""
+DIAMOND PROBLEM
```

Comment:
LTGM 
Only comment not related to task: PEP 8: E302 expected 2 blank lines, found 1